packages/taglib.rb

```diff
@@ -3,35 +3,35 @@
 class Taglib < Package
```

Comment:
`class Taglib < CMake` with sed in patch section?

Reply:
wasn't working. hence the non-standard build options.